### ISSUE TITLE
Keep displaying the chiventure logo until we have something better to show

### ISSUE THREAD

**Issue by borjasotomayor** [uchicago-cs/chiventure#405]
Right now, chiventure shows up this logo on startup:

```
┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│              ________________________________________________________________________________________               │
│            / \                                                                                       \              │
│           |   |                                                                                      |              │
│            \_ |     ██████╗██╗  ██╗██╗██╗   ██╗███████╗███╗   ██╗████████╗██╗   ██╗██████╗ ███████╗  |              │
│               |    ██╔════╝██║  ██║██║██║   ██║██╔════╝████╗  ██║╚══██╔══╝██║   ██║██╔══██╗██╔════╝  |              │
│               |    ██║     ███████║██║██║   ██║█████╗  ██╔██╗ ██║   ██║   ██║   ██║██████╔╝█████╗    |              │
│               |    ██║     ██╔══██║██║╚██╗ ██╔╝██╔══╝  ██║╚██╗██║   ██║   ██║   ██║██╔══██╗██╔══╝    |              │
│               |    ╚██████╗██║  ██║██║ ╚████╔╝ ███████╗██║ ╚████║   ██║   ╚██████╔╝██║  ██║███████╗  |              │
│               |     ╚═════╝╚═╝  ╚═╝╚═╝  ╚═══╝  ╚══════╝╚═╝  ╚═══╝   ╚═╝    ╚═════╝ ╚═╝  ╚═╝╚══════╝  |              │
│               |   ___________________________________________________________________________________|___           │
│               |  /                                                                                      /           │
│               \_/______________________________________________________________________________________/            │
│                                                                                                                     │
│                                                                                                                     │
│                                            Type 'HELP' to show help menu                                            │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

But the moment you press a key, it switches to this:

```
┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Main Window                                                                                                         │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
│                                                                                                                     │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

Eventually, we will have something useful to show there (graphics, the map, etc.) but, until then, we should just keep displaying the chiventure logo. What we may want to do is to make the `Type 'HELP' to show help menu` message disappear once the first keypress happens.

**Comment by ecoble** [comment]
I have made the changes to continue displaying the banner in the top screen, and to remove the help message once the player starts typing. However, when you toggle to map mode, every time you press enter after a command it flashes back to the banner briefly and then back to the map (looks really glitchy). I think this was always an issue but wasn't very noticeable before when the top screen was mostly blank, I'll try to fix this.

**Comment by borjasotomayor** [comment]
Closed by #407
